A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29


a. Build 99% Confidence Interval Using Sample Standard Deviation
Assuming the sample is representative of the population, construct a 99% confidence interval for the mean number of characters printed before the print-head fails using the sample standard deviation. Explain the steps you take and the rationale behind using the t-distribution for this task.


In [24]:
import pandas as pd
import numpy as np
import scipy.stats as stats
import math

In [2]:
data=[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

In [15]:
stats.t.interval( df=15, loc=np.mean(data), scale=stats.sem(data),confidence=0.99) 


(15.39595613775819, 25.404043862241807)

the interval is (15.39595613775819, 25.404043862241807)

b. Build 99% Confidence Interval Using Known Population Standard Deviation
If it were known that the population standard deviation is 0.2 million characters, construct a 99% confidence interval for the mean number of characters printed before failure.


confidence interval =(sample mean+ margin of error,sample mean-margin of error)

In [22]:
pop_std=0.2
sample_mean=np.mean(data)
sample_mean


20.4

margin of error = T crictical *(population std/sqrt(sample size))

In [21]:
T_crictical=stats.t.ppf(1-0.01/2,14)
T_crictical

2.976842734370834

In [26]:
Margin_of_Error=T_crictical*(0.2/math.sqrt(15))
Margin_of_Error

0.15372349779329053

In [27]:
sample_mean+Margin_of_Error

20.55372349779329

In [28]:
sample_mean-Margin_of_Error

20.246276502206708

the interval is(20.246276502206708,20.55372349779329)